### PR TITLE
1h Maces General Speed Reduction

### PR DESCRIPTION
**Pull request by Telfordski**
Work in Progress (Done). 

https://docs.google.com/spreadsheets/d/1UsgIDQG861FlnY_wKnLKQfcSnu2zxiNp1x2k6bIIGb4/edit#gid=2132882987

### CODE REVIEW

**Comment by Yeldur** [comment]
Converting to draft as not ready for review yet

**Comment by Telfordski** [comment]
> Converting to draft as not ready for review yet

ty should have it done by end of week

**Comment by Telfordski** [comment]
> Converting to draft as not ready for review yet

Think that's done. Finally.

File: items/weapons.xml [hunk 1]
```diff
@@ -4084,108 +4084,108 @@
     </ItemComponent>
     <Flags ForceAttachOffHandPrimaryItemBone="true" />
   </Item>
-  <CraftedItem id="crpg_steel_flanged_mace_v1_h0" name="{=fz23ajFz}Steel Flanged Mace" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
+  <CraftedItem id="crpg_steel_flanged_mace_v3_h0" name="{=fz23ajFz}Steel Flanged Mace" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
```
Comment (Yeldur) [inline items/weapons.xml:4087]: Should go to v2 not v3

File: items/weapons.xml [hunk 2]
```diff
@@ -4084,108 +4084,108 @@
...
-  <CraftedItem id="crpg_steel_flanged_mace_v1_h0" name="{=fz23ajFz}Steel Flanged Mace" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
+  <CraftedItem id="crpg_steel_flanged_mace_v3_h0" name="{=fz23ajFz}Steel Flanged Mace" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
     <Pieces>
       <Piece id="crpg_steel_flanged_mace_blade_h0" Type="Blade" />
       <Piece id="crpg_steel_flanged_mace_handle_h0" Type="Handle" scale_factor="88" />
       <Piece id="crpg_steel_flanged_mace_pommel_h0" Type="Pommel" scale_factor="90" />
     </Pieces>
   </CraftedItem>
-  <CraftedItem id="crpg_steel_flanged_mace_v1_h1" name="{=fz23ajFz}Steel Flanged Mace +1" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
+  <CraftedItem id="crpg_steel_flanged_mace_v3_h1" name="{=fz23ajFz}Steel Flanged Mace +1" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
```
Comment (Yeldur) [inline items/weapons.xml:4094]: Should go to v2 not v3

File: items/weapons.xml [hunk 3]
```diff
@@ -4084,108 +4084,108 @@
...
     <Pieces>
       <Piece id="crpg_steel_flanged_mace_blade_h1" Type="Blade" />
       <Piece id="crpg_steel_flanged_mace_handle_h1" Type="Handle" scale_factor="88" />
       <Piece id="crpg_steel_flanged_mace_pommel_h1" Type="Pommel" scale_factor="90" />
     </Pieces>
   </CraftedItem>
-  <CraftedItem id="crpg_steel_flanged_mace_v1_h2" name="{=fz23ajFz}Steel Flanged Mace +2" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
+  <CraftedItem id="crpg_steel_flanged_mace_v3_h2" name="{=fz23ajFz}Steel Flanged Mace +2" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
```
Comment (Yeldur) [inline items/weapons.xml:4101]: Should go to v2 not v3

File: items/weapons.xml [hunk 4]
```diff
@@ -4084,108 +4084,108 @@
...
     <Pieces>
       <Piece id="crpg_steel_flanged_mace_blade_h2" Type="Blade" />
       <Piece id="crpg_steel_flanged_mace_handle_h2" Type="Handle" scale_factor="88" />
       <Piece id="crpg_steel_flanged_mace_pommel_h2" Type="Pommel" scale_factor="90" />
     </Pieces>
   </CraftedItem>
-  <CraftedItem id="crpg_steel_flanged_mace_v1_h3" name="{=fz23ajFz}Steel Flanged Mace +3" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
+  <CraftedItem id="crpg_steel_flanged_mace_v3_h3" name="{=fz23ajFz}Steel Flanged Mace +3" crafting_template="crpg_Mace" culture="Culture.khuzait" modifier_group="mace">
```
Comment (Yeldur) [inline items/weapons.xml:4108]: Should go to v2 not v3

File: items/weapons.xml [hunk 5]
```diff
@@ -10340,105 +10340,105 @@
...
-  <CraftedItem id="crpg_militia_pernach_v1_h3" name="{=WASZXuec}Militia Pernach +3" crafting_template="crpg_Mace" is_merchandise="false">
+  <CraftedItem id="crpg_militia_pernach_v2_h3" name="{=WASZXuec}Militia Pernach +3" crafting_template="crpg_Mace" is_merchandise="false">
     <Pieces>
       <Piece id="crpg_militia_pernach_blade_h3" Type="Blade" scale_factor="137" />
       <Piece id="crpg_militia_pernach_handle_h3" Type="Handle" scale_factor="142" />
       <Piece id="crpg_militia_pernach_pommel_h3" Type="Pommel" scale_factor="90" />
     </Pieces>
   </CraftedItem>
-  <CraftedItem id="crpg_heavy_flanged_mace_v1_h0" name="{=mzvtFPPi}Heavy Flanged Mace" crafting_template="crpg_Mace" is_merchandise="false">
+  <CraftedItem id="crpg_heavy_flanged_mace_v3_h0" name="{=mzvtFPPi}Heavy Flanged Mace" crafting_template="crpg_Mace" is_merchandise="false">
```
Comment (Yeldur) [inline items/weapons.xml:10423]: Should go to v2 not v3

File: items/weapons.xml [hunk 6]
```diff
@@ -10340,105 +10340,105 @@
...
     <Pieces>
       <Piece id="crpg_heavy_flanged_mace_blade_h0" Type="Blade" scale_factor="100" />
       <Piece id="crpg_heavy_flanged_mace_handle_h0" Type="Handle" scale_factor="95" />
     </Pieces>
   </CraftedItem>
-  <CraftedItem id="crpg_heavy_flanged_mace_v1_h1" name="{=mzvtFPPi}Heavy Flanged Mace +1" crafting_template="crpg_Mace" is_merchandise="false">
+  <CraftedItem id="crpg_heavy_flanged_mace_v3_h1" name="{=mzvtFPPi}Heavy Flanged Mace +1" crafting_template="crpg_Mace" is_merchandise="false">
```
Comment (Yeldur) [inline items/weapons.xml:10429]: Should go to v2 not v3

File: items/weapons.xml [hunk 7]
```diff
@@ -10340,105 +10340,105 @@
...
     <Pieces>
       <Piece id="crpg_heavy_flanged_mace_blade_h1" Type="Blade" scale_factor="100" />
       <Piece id="crpg_heavy_flanged_mace_handle_h1" Type="Handle" scale_factor="95" />
     </Pieces>
   </CraftedItem>
-  <CraftedItem id="crpg_heavy_flanged_mace_v1_h2" name="{=mzvtFPPi}Heavy Flanged Mace +2" crafting_template="crpg_Mace" is_merchandise="false">
+  <CraftedItem id="crpg_heavy_flanged_mace_v3_h2" name="{=mzvtFPPi}Heavy Flanged Mace +2" crafting_template="crpg_Mace" is_merchandise="false">
```
Comment (Yeldur) [inline items/weapons.xml:10435]: Should go to v2 not v3

File: items/weapons.xml [hunk 8]
```diff
@@ -10340,105 +10340,105 @@
...
     <Pieces>
       <Piece id="crpg_heavy_flanged_mace_blade_h2" Type="Blade" scale_factor="100" />
       <Piece id="crpg_heavy_flanged_mace_handle_h2" Type="Handle" scale_factor="95" />
     </Pieces>
   </CraftedItem>
-  <CraftedItem id="crpg_heavy_flanged_mace_v1_h3" name="{=mzvtFPPi}Heavy Flanged Mace +3" crafting_template="crpg_Mace" is_merchandise="false">
+  <CraftedItem id="crpg_heavy_flanged_mace_v3_h3" name="{=mzvtFPPi}Heavy Flanged Mace +3" crafting_template="crpg_Mace" is_merchandise="false">
```
Comment (Yeldur) [inline items/weapons.xml:10441]: Should go to v2 not v3

File: items.json [hunk 9]
```diff
@@ -2764,8 +2764,8 @@
     ]
   },
   {
-    "id": "crpg_arabian_mace_v1_h0",
-    "baseId": "crpg_arabian_mace_v1",
+    "id": "crpg_arabian_mace_v2_h0",
```
Comment (Yeldur) [inline items.json:2767]: No stat changes, this shouldn't be refunded

File: items.json [hunk 10]
```diff
@@ -60799,9 +60799,9 @@
     "name": "Flanged Mace",
     "culture": "Aserai",
     "type": "OneHandedWeapon",
-    "price": 6664,
-    "weight": 1.81,
-    "tier": 9.730039,
+    "price": 4313,
+    "weight": 1.9,
+    "tier": 7.60975,
```
Comment (Yeldur) [inline items.json:60804]: Tier reduction, needs a refund

File: items.json [hunk 11]
```diff
@@ -109648,14 +109648,14 @@
     ]
   },
   {
-    "id": "crpg_military_hammer_v1_h0",
-    "baseId": "crpg_military_hammer_v1",
+    "id": "crpg_military_hammer_v2_h0",
```
Comment (Yeldur) [inline items.json:109651]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 12]
```diff
@@ -109789,21 +109789,21 @@
...
+        "swingDamage": 27,
         "swingDamageType": "Blunt",
         "swingSpeed": 86
       }
     ]
   },
   {
-    "id": "crpg_militia_pernach_v1_h0",
-    "baseId": "crpg_militia_pernach_v1",
+    "id": "crpg_militia_pernach_v2_h0",
```
Comment (Yeldur) [inline items.json:109799]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 13]
```diff
@@ -104342,14 +104342,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_long_pernach_v1_h0",
-    "baseId": "crpg_long_pernach_v1",
+    "id": "crpg_long_pernach_v2_h0",
```
Comment (Yeldur) [inline items.json:104345]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

**Review by Yeldur — CHANGES_REQUESTED** [review]
There are a few changes that need to be made but the glaring issue right now is primarily that this change results in massive tier reductions across the board for 1h mace but does not offer anything to tier-match them to their old tiers.

What we do want is for speed to be reduced

What we don't want is to have zero high tier maces, which this change would seemingly enforce based on what I can see from the changes.

General rule of thumb I would say is that super high speed maces like 88+ speed are super fast and should receive a speed reduction **BUT** be given damage in return for their lost speed to ensure that tier changes are not major.

I've added three review points marked as:
"example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically." to give an example of some items that shouldn't really see a speed reduction - There ARE more examples of this though and will need you to address them before approval.

TL;DR

Where we reduce stats, we should increase others to ensure that tier does not change drastically

File: items.json [hunk 14]
```diff
@@ -30698,14 +30698,14 @@
...
+    "baseId": "crpg_byzantine_mace_v2",
     "name": "Byzantine Mace",
     "culture": "Empire",
     "type": "OneHandedWeapon",
-    "price": 5384,
-    "weight": 1.82,
-    "tier": 8.632379,
+    "price": 3517,
+    "weight": 1.92,
+    "tier": 6.763587,
```
Comment (Yeldur) [inline items.json:30708]: speed reduction is OK but should be offset by an increase in damage to match tiers

File: items.json [hunk 15]
```diff
@@ -30822,30 +30822,30 @@
...
+    "baseId": "crpg_calradic_mace_v2",
     "name": "Calradic Mace",
     "culture": "Empire",
     "type": "OneHandedWeapon",
-    "price": 6281,
-    "weight": 1.72,
-    "tier": 9.413677,
+    "price": 4069,
+    "weight": 1.81,
+    "tier": 7.35880756,
```
Comment (Yeldur) [inline items.json:30848]: speed reduction is OK but should be offset by an increase in damage to match tiers

File: items.json [hunk 16]
```diff
@@ -49246,14 +49246,14 @@
...
+    "baseId": "crpg_eastern_heavy_mace_v2",
     "name": "Eastern Heavy Mace",
     "culture": "Khuzait",
     "type": "OneHandedWeapon",
-    "price": 3728,
-    "weight": 2.04,
-    "tier": 6.99636,
+    "price": 2948,
+    "weight": 2.12,
+    "tier": 6.09928036,
```
Comment (Yeldur) [inline items.json:49256]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 17]
```diff
@@ -59062,19 +59062,19 @@
...
         "swingDamageType": "Blunt",
-        "swingSpeed": 77
+        "swingSpeed": 76
       }
     ]
   },
   {
-    "id": "crpg_fine_steel_mace_v1_h3",
-    "baseId": "crpg_fine_steel_mace_v1",
+    "id": "crpg_fine_steel_mace_v2_h3",
```
Comment (Yeldur) [inline items.json:59070]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 18]
```diff
@@ -62972,14 +62972,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_fullered_western_mace_v1_h0",
-    "baseId": "crpg_fullered_western_mace_v1",
+    "id": "crpg_fullered_western_mace_v2_h0",
```
Comment (Yeldur) [inline items.json:62975]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 19]
```diff
@@ -73396,14 +73396,14 @@
     ]
   },
   {
-    "id": "crpg_heavy_horsemans_mace_v1_h0",
-    "baseId": "crpg_heavy_horsemans_mace_v1",
+    "id": "crpg_heavy_horsemans_mace_v2_h0",
```
Comment (Yeldur) [inline items.json:73399]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 20]
```diff
@@ -89036,14 +89036,14 @@
     ]
   },
   {
-    "id": "crpg_judgement_v1_h0",
-    "baseId": "crpg_judgement_v1",
+    "id": "crpg_judgement_v2_h0",
```
Comment (Yeldur) [inline items.json:89039]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 21]
```diff
@@ -93240,14 +93240,14 @@
     ]
   },
   {
-    "id": "crpg_knights_fall_v1_h0",
-    "baseId": "crpg_knights_fall_v1",
+    "id": "crpg_knights_fall_v2_h0",
```
Comment (Yeldur) [inline items.json:93243]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 22]
```diff
@@ -102854,14 +102854,14 @@
     ]
   },
   {
-    "id": "crpg_long_cavalry_mace_v1_h0",
-    "baseId": "crpg_long_cavalry_mace_v1",
+    "id": "crpg_long_cavalry_mace_v2_h0",
```
Comment (Yeldur) [inline items.json:102857]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 23]
```diff
@@ -108784,14 +108784,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_medium_goedendag_v1_h0",
-    "baseId": "crpg_medium_goedendag_v1",
+    "id": "crpg_medium_goedendag_v2_h0",
```
Comment (Yeldur) [inline items.json:108787]: speed reduction is OK but should be offset by an increase in damage to match tiers

File: items.json [hunk 24]
```diff
@@ -128252,14 +128252,14 @@
     ]
   },
   {
-    "id": "crpg_onehanded_barmace_v1_h0",
-    "baseId": "crpg_onehanded_barmace_v1",
+    "id": "crpg_onehanded_barmace_v2_h0",
```
Comment (Yeldur) [inline items.json:128255]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 25]
```diff
@@ -128387,19 +128387,19 @@
...
         "swingDamageType": "Blunt",
-        "swingSpeed": 87
+        "swingSpeed": 86
       }
     ]
   },
   {
-    "id": "crpg_onehanded_gada_v1_h0",
-    "baseId": "crpg_onehanded_gada_v1",
+    "id": "crpg_onehanded_gada_v2_h0",
```
Comment (Yeldur) [inline items.json:128395]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 26]
```diff
@@ -131664,14 +131664,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_persian_steel_mace_v1_h0",
-    "baseId": "crpg_persian_steel_mace_v1",
+    "id": "crpg_persian_steel_mace_v2_h0",
```
Comment (Yeldur) [inline items.json:131667]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 27]
```diff
@@ -159766,14 +159766,14 @@
     ]
   },
   {
-    "id": "crpg_spiralled_mace_v1_h0",
-    "baseId": "crpg_spiralled_mace_v1",
+    "id": "crpg_spiralled_mace_v2_h0",
```
Comment (Yeldur) [inline items.json:159769]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

File: items.json [hunk 28]
```diff
@@ -164080,14 +164080,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_steppe_spiked_mace_v1_h0",
-    "baseId": "crpg_steppe_spiked_mace_v1",
+    "id": "crpg_steppe_spiked_mace_v2_h0",
```
Comment (Yeldur) [inline items.json:164083]: speed reduction is OK but should be offset by an increase in damage to match tiers

File: items.json [hunk 29]
```diff
@@ -186544,14 +186544,14 @@
     ]
   },
   {
-    "id": "crpg_warhammer_v1_h0",
-    "baseId": "crpg_warhammer_v1",
+    "id": "crpg_warhammer_v2_h0",
```
Comment (Yeldur) [inline items.json:186547]: example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.

**Review by Yeldur — CHANGES_REQUESTED** [review]
Cont from above

**Comment by Yeldur** [comment]
Following discussion am moving this back to draft temporarily whilst discussion is had with balance team over maces.

**Comment by Telfordski** [comment]
> There are a few changes that need to be made but the glaring issue right now is primarily that this change results in massive tier reductions across the board for 1h mace but does not offer anything to tier-match them to their old tiers.
> 
> What we do want is for speed to be reduced
> 
> What we don't want is to have zero high tier maces, which this change would seemingly enforce based on what I can see from the changes.
> 
> General rule of thumb I would say is that super high speed maces like 88+ speed are super fast and should receive a speed reduction **BUT** be given damage in return for their lost speed to ensure that tier changes are not major.
> 
> I've added a few points marked as: "example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically." to give an example of some items that shouldn't really see a speed reduction.
> 
> TL;DR
> 
> Where we reduce stats, we should increase others to ensure that tier does not change drastically

that's just like your opinion man - at least I assume that's what you're basing the assessments of these reductions on, for the ones you passed comment.

I don't want to match their old tiers, or swap damage for speed; I want to reduce the speed of the maces I reduced - based on their damage and length and overall options available. I want to reduce the speed, for the damage and length they have already - feel free to tweak them yourself / rebuff or revert afterwards if you think the nerfs were unwelcome, but let's quickly sort out this budget adjustment at least, because no-one can really assess shit with these broken ass numbers - like some crazy scientist I can create some absolutely broken unfathomable shit, and it's my own assessments and testing that have prevented that instead of the budget system - even that said, some are still potentially too fast, but I want to experiment with handling more first before any more changes. Most of the maces available are fairly high tier options (at least tier 8 in terms of meta), because I'm a shit crafter who just wants to make viable ass options for everyone with good trade-offs. If the budget correction isn't perfect, so be it, perhaps it can be weighted more towards speed, but fundamentally it must be adapted and I think the current set of maces (although I'm biased) are a good place to start because I've made them at their perceived limits.

**Comment by Yeldur** [comment]
Closing as budget changes have since come in to adjust 1h blunt rendering this PR null & void
